explore/src/main/scala/explore/targeteditor/AladinCell.scala

```diff
@@ -230,29 +226,28 @@ object AladinCell extends ModelOptics with AladinCommon:
       // target options, will be read from the user preferences
       .useStateView(Pot.pending[AsterismVisualOptions])
       // to get faster reusability use a serial state, rather than check every candidate
```

Comment:
Should the `serial state` comment be removed, too?

Reply:
Yeah, it is removed in my next PR